wsgi.py

```diff
@@ -5,4 +5,4 @@
 
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0")
+    app.run()
```

Comment:
Instead of removing the `host` we could make it a parameter and pass it in using `click` or `argparse`.